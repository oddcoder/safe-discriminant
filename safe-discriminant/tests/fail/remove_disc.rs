use safe_discriminant::Discriminant;
use safe_discriminant_derive::remove_disc;
#[derive(Discriminant)]
#[repr(u8)]
pub enum Foo {
    #[remove_disc]
    A = 0,
    B = 1,
}

fn main() {}
