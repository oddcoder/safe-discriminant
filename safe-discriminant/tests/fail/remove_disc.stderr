error: expected non-macro attribute, found attribute macro `remove_disc`
 --> tests/fail/remove_disc.rs:6:7
  |
6 |     #[remove_disc]
  |       ^^^^^^^^^^^ not a non-macro attribute
